public/video-ui/src/components/utils/YouTubeEmbed.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {getStore} from '../../util/storeAccessor';
+
+export function youTubeUrl(id) {
+    return `https://www.youtube.com/watch?v=${id}`;
```

Comment:
[YouTube recommend](https://developers.google.com/youtube/player_parameters#Manual_IFrame_Embeds) using the `https://www.youtube.com/embed/VIDEO_ID` format.